### PR TITLE
Linux Bash Easy installer script

### PR DESCRIPTION
**Pull request by helloitu**
Short Description: 
 This very simple script easy configure the bot for Noob users.

Fixes:
- 
- 
-

### CODE REVIEW

File: easy-installer.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```
Comment (solderzzc) [inline easy-installer.sh]: Should be renamed to 'Linux' ?

**Comment by solderzzc** [comment]
easy-installer.sh should rename to linux-easy-installer.

**Comment by nikhil-pandey** [comment]
Why not use the default config and replace the values there using `sed`? This way will have to maintain additional config.

**Comment by solderzzc** [comment]
Is that better to work out a combine version with this pr
https://github.com/PokemonGoF/PokemonGo-Bot/pull/3263

**Comment by simonsmh** [comment]
Of course sed is much better. #3263

**Comment by helloitu** [comment]
okay, congratulations on arrogance ^^, closed request.

**Pull request closed without merging**